website/docs/guides/flagger-manual-gating.mdx

```diff
@@ -0,0 +1,147 @@
+---
+title: Manual approval for progressive delivery deployments
+sidebar_position: 6
+hide_title: true
+---
+
+import TierLabel from "../_components/TierLabel";
+
+<h1>
+  {frontMatter.title} <TierLabel tiers="Enterprise" />
+</h1>
+
+Weave GitOps Enterprise helps you understand the state of progressive delivery 
+updates to your applications with [Flagger](https://flagger.app). The Delivery 
+view shows all your deployed `Canary` objects and the status for how a rollout 
+is progressing. 
+
+By default, Flagger will automatically promote a new version of an application 
+should it pass the defined checks during an analysis phase. However, you can 
+also configure [webhooks](https://docs.flagger.app/usage/webhooks) to enable 
+manual approvals for Flagger to proceed to the next phase of a rollout.
+
+In this guide we will show you how to get started with manually gating a 
+progressive delivery promotion with Flagger, using the in-built load tester as 
+a way to demonstrate and learn the capability so that you could configure 
+your own gates.
+
+## Pre-requisites
+- Basic knowledge of [Flagger](https://flagger.app).
+- An existing `Canary` object and target deployment.
+- Flagger's load tester [installed](https://docs.flagger.app/usage/webhooks#load-testing)
+
+## Basic introduction to Webhooks and Gating
+Flagger can be configured to work with several types of hooks which will be called at 
+given stages during a progressive delivery rollout. Some of these allow you to manually 
+gate whether a rollout proceeds at certain points:
+- Before a new deployment is scaled up and canary analysis begins with `confirm-rollout`. 
+- Before traffic weight is increased with `confirm-traffic-increase`.
+- Before a new version is promoted following successful canary analysis with `confirm-promotion`.
+
+Any URL can be used as a webhook target, if will approve if it returns with a
```

Comment:
might typo with  `if` ? `if will approve if it returns with` might be `it will approve if it returns with`

Reply:
good catch